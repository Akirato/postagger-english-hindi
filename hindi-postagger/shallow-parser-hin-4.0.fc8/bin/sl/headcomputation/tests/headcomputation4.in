<Sentence id="1">
1	((	NP	
1.1	himAlayZa	NNP	<fs af='himAlayZa,n,,sg,,d,0,0'>
1.2	parbawamAlAra	NN	<fs af='parbawamAlA,n,,sg,,o,era,era'>
	))		
2	((	NP	
2.1	abarNanIyZa	JJ	<fs af='abarNanIyZa,adj,,,,,,'>
	))		
3	((	CCP	
3.1	o	CC	<fs af='o,avy,,,,,,'>
	))		
4	((	NP	
4.1	nAtakIyZa	JJ	<fs af='nAtakIyZa,adj,,,,,,'>
4.2	xqSya	NN	<fs af='xqSya,n,,sg,,d,0,0'>
	))		
5	((	VGNN	
5.1	xeKAra	VM	<fs af='xeK,n,,,any,,,' poslcat="NM">
	))		
6	((	NP	
6.1	janya	PSP	<fs af='janya,psp,,,,,,'>
	))		
7	((	NP	
7.1	uwwara	NN	<fs af='uwwara,n,,sg,,d,0,0'>
	))		
8	((	NP	
8.1	sikimera	NN	<fs af='sikima,n,,sg,,o,era,era'>
	))		
9	((	NP	
9.1	wulanA	NN	<fs af='wulanA,n,,sg,,d,0,0'>
	))		
10	((	VGF	
10.1	hayZa	VM	<fs af='ha,v,,,5,,e,e'>
10.2	nA	NEG	<fs af='nA,avy,,,,,,'>
10.3	.	SYM	<fs af='.,punc,,,,,,' poslcat="NM">
	))		
</Sentence>
