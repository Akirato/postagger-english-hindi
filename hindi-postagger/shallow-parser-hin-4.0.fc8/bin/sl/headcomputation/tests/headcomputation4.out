<Sentence id="1">
1	((	NP	<fs af='parbawamAlA,n,,sg,,o,era,era' head=parbawamAlAra>
1.1	himAlayZa	NNP	<fs af='himAlayZa,n,,sg,,d,0,0' name=himAlayZa>
1.2	parbawamAlAra	NN	<fs af='parbawamAlA,n,,sg,,o,era,era' name=parbawamAlAra>
	))		
2	((	NP	<fs af='abarNanIyZa,adj,,,,,,' head=abarNanIyZa>
2.1	abarNanIyZa	JJ	<fs af='abarNanIyZa,adj,,,,,,' name=abarNanIyZa>
	))		
3	((	CCP	<fs af='o,avy,,,,,,' head=o>
3.1	o	CC	<fs af='o,avy,,,,,,' name=o>
	))		
4	((	NP	<fs af='xqSya,n,,sg,,d,0,0' head=xqSya>
4.1	nAtakIyZa	JJ	<fs af='nAtakIyZa,adj,,,,,,' name=nAtakIyZa>
4.2	xqSya	NN	<fs af='xqSya,n,,sg,,d,0,0' name=xqSya>
	))		
5	((	VGNN	<fs af='xeK,n,,,any,,,' poslcat="NM" head=xeKAra>
5.1	xeKAra	VM	<fs af='xeK,n,,,any,,,' name=xeKAra poslcat="NM">
	))		
6	((	NP	<fs af='janya,psp,,,,,,' head=janya>
6.1	janya	PSP	<fs af='janya,psp,,,,,,' name=janya>
	))		
7	((	NP	<fs af='uwwara,n,,sg,,d,0,0' head=uwwara>
7.1	uwwara	NN	<fs af='uwwara,n,,sg,,d,0,0' name=uwwara>
	))		
8	((	NP	<fs af='sikima,n,,sg,,o,era,era' head=sikimera>
8.1	sikimera	NN	<fs af='sikima,n,,sg,,o,era,era' name=sikimera>
	))		
9	((	NP	<fs af='wulanA,n,,sg,,d,0,0' head=wulanA>
9.1	wulanA	NN	<fs af='wulanA,n,,sg,,d,0,0' name=wulanA>
	))		
10	((	VGF	<fs af='ha,v,,,5,,e,e' head=hayZa>
10.1	hayZa	VM	<fs af='ha,v,,,5,,e,e' name=hayZa>
10.2	nA	NEG	<fs af='nA,avy,,,,,,' name=nA>
10.3	.	SYM	<fs af='.,punc,,,,,,' name=. poslcat="NM">
	))		
</Sentence>
